CHANGELOG.unreleased.md

```diff
@@ -28,4 +28,5 @@
 >
 > In case of conflict, the highest (lowest in previous list) `$version` wins.
 
+- @xen-orchestra/fs patch
```

Comment:
We need to re-release `@xen-orchestra/proxy` and `xo-server` to ship this fix, please add them too :slightly_smiling_face: